Makefile

```diff
@@ -43,8 +43,8 @@ deploy: manifests
 	cd config/default && if [ ${KFSERVING_ENABLE_SELF_SIGNED_CA} != false ]; then \
 	kustomize edit remove resource certmanager/certificate.yaml; \
 	else kustomize edit add resource certmanager/certificate.yaml; fi;
-	kubectl get crd inferenceservices.serving.kubeflow.org && kustomize build config/default/crd | kubectl replace --validate=false -f - || \
-	kustomize build config/default/crd | kubectl create --validate=false -f -
+	kubectl get crd inferenceservices.serving.kubeflow.org && kustomize build config/crd | kubectl replace --validate=false -f - || \
```

Comment:
if we are going to apply the crd below, there's no purpose for create/replace the crd over here.

Reply:
Should be fine to just remove the added lines here and just keep the `kustomize build config/default | kubectl apply --validate=false -f -` since that  will now install the crd, right?

Reply:
yes, this replace/create command is for replacing the crd without storing the crd spec in the annotations field. However, since we put the crd in `config/default`, we will be always applying the crd with annotations.